src/main/java/roomescape/repository/repository/ReservationRepository.java

```diff
@@ -0,0 +1,66 @@
+package roomescape.repository.repository;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import roomescape.model.Reservation;
+import roomescape.model.ReservationTime;
+import roomescape.repository.Dao;
+
+@org.springframework.stereotype.Repository
```

Comment:
Repository 추상클래스가 별도 정의되어 이렇게밖에 표현이 안되는데요!

클래스명을 고민할 때 프레임워크에서 제공하는 + 빈번하게 사용되는 클래스명은 피하는 것이 좋습니다.
Repository 추상클래스 이름을 변경해주세요 😄 

```suggestion
@Repository
```

Reply:
프레임워크에서 제공하는 기본 클래스명과 겹치지 않도록 하는 게 확실히 좋겠네요!
수정하겠습니다. 감사합니다!

[rename: 클래스명 변경(Repository -> DBRepository)](https://github.com/woowacourse/spring-roomescape-admin/pull/324/commits/449306cc31495a8ebd3bb66a2cfa4702e75ee397)